test/news.js

```diff
@@ -0,0 +1,118 @@
+const assert = require('assert');
```

Comment:
Let's add more details to the test file name. Like when you are testing a controller you can write 
controllers.news.js name. And when it's a service test it can be services.player.js